lib/schema_plus/core/sql_struct.rb

```diff
@@ -20,9 +25,10 @@ def parse!(sql)
           self.name = m[:name]
           self.body = m[:body]
           self.options = m[:options]
+          self.inheritance = m[:inheritance] if m.names.include? INHERITANCE
```

Comment:
...and with the optional term in the regex, the `if names.include?` guard won't be needed:

```ruby
self.inheritance = m[:inheritance]
```